mods/farming/api.lua

```diff
@@ -213,16 +178,15 @@ farming.register_plant = function(name, def)
 
 	-- Register growing steps
 	for i=1,def.steps do
```

Comment:
You should fix the code around your changes, as you're touching most of the file.